packages/api/blocks-api/src/blocks/createRichTextBlock.ts

```diff
@@ -70,12 +71,10 @@ export interface RichTextBlockInputInterface<LinkBlockInput extends BlockInputIn
 export function createRichTextBlock<LinkBlock extends Block>({
     link: LinkBlock,
     indexSearchText = true,
+    migrateOptions = { migrations: [], version: 0 },
```

Comment:
In our other factories we use `nameOrOptions` as second argument, which also allows overriding the block's name: https://github.com/vivid-planet/comet/blob/main/packages/api/blocks-api/src/blocks/createTextLinkBlock.ts#L31. I'd prefer to keep this consistent across the code base